src/lucky/redirectable.cr

```diff
@@ -1,19 +1,44 @@
+# Redirect the browser
+#
+# There are multiple ways to redirect inside of an action. The most common ways are to redirect to a `Lucky::Action` class, or a URL/path `String`. Both use the `redirect` method:
+#
+# ```crystal
+# redirect to: Users::Index
+# redirect to: "https://luckyframework.org/"
+# redirect to: "/users"
```

Comment:
I'm thinking of removing the `to` and recommending people pass just the string or action: `redirect Users::Index` thoughts?

Reply:
I kind of like the `to` to be honest, but I can't say I have a strong argument _for_ it. I've never removed explicitness and been sad later. It's also pretty short, and I like when methods can read more like english.

Also, you don't _need_ to use it because of how crystal handles arguments:

```cr
require "./src/lucky"
require "./spec/spec_helper"
include ContextHelper

class Users::Index < Lucky::Action
  get "/users" do
    text "you won't see me"
  end
end

index = Users::Index.new(build_context, params)
route = Lucky::RouteHelper.new(method: :get, path: "/users")

index.redirect(route).context.response.headers
index.redirect(to: route).context.response.headers
```